recipe/meta.yaml

```diff
@@ -38,16 +38,17 @@ test:
     - humanfriendly --help  # [not unix]
 
 about:
-  home: https://humanfriendly.readthedocs.org
+  home: https://humanfriendly.readthedocs.io
   license: MIT
   license_family: MIT
   license_file: LICENSE.txt
+  license_url: https://github.com/xolox/python-humanfriendly/blob/master/LICENSE.txt
```

Comment:
the license file is in the package. no need (even dangerous in terms of correctness) to add here this superflous license_url on top. please stop this habbit